gui/bar.py

```diff
@@ -0,0 +1,15 @@
+import pygame
+
+
+class Bar:
+    def __init__(self, x, y, max_points, colour_of_background_bar, colour_of_front_bar):
+        self.x = x
+        self.y = y
+        self.max_points = max_points
```

Comment:
I prefer to delete max_points from __init__ and add it in draw method instead as a argument. It will be better beacuse when i will import Bar method in to my gui.screen player/enemy instance will be in other file. That's will make no way to use it corretly.